.github/PULL_REQUEST_TEMPLATE/new_petab_problem.md

```diff
@@ -0,0 +1,11 @@
+# Checklist
+- [ ] Information about the model is in the SBML metadata
+    - [ ] Reference to the original paper, and any other relevant references
+    - [ ] A brief model description (one or two sentences)
+    - [ ] A brief description of the data (one or two sentences)
+    - [ ] Any differences between the implementation and the original publication
+- [ ] Information is in the model's GitHub issue
+    - [ ] Any differences between the implementation and the original publication
+    - [ ] Experience of fitting / uncertainty analysis (e.g. optimizer used, hyperparameters)
```

Comment:
"how `nominalParameters` were obtained"? I  guess it's in there, but might be good to be more explicit?